### PR TITLE
Only request STORAGE_BINDING for store cases

### PR DESCRIPTION
**Pull request by jchen10**
Issue: #915 

<hr>

**Requirements for PR author:**

- [x] All missing test coverage is tracked with "TODO" or `.unimplemented()`.
- [x] New helpers are `/** documented */` and new helper files are found in `helper_index.txt`.
- [x] Test behaves as expected in a WebGPU implementation. (If not passing, explain above.)

**Requirements for [reviewer sign-off](https://github.com/gpuweb/cts/blob/main/docs/reviews.md):**

- [ ] Tests are properly located in the test tree.
- [ ] [Test descriptions](https://github.com/gpuweb/cts/blob/main/docs/intro/plans.md) allow a reader to "read only the test plans and evaluate coverage completeness", and accurately reflect the test code.
- [ ] Tests provide complete coverage (including validation control cases). **Missing coverage MUST be covered by TODOs.**
- [ ] Helpers and types promote readability and maintainability.

When landing this PR, be sure to make any necessary issue status updates.

### CODE REVIEW

**Comment by jchen10** [comment]
So that we can still run 'draw' and 'copy' cases on platforms where STORAGE_BINDING is not available yet.

File: src/webgpu/web_platform/reftests/canvas_complex.html.ts [hunk 1]
```diff
@@ -697,12 +697,18 @@ fn main(@builtin(global_invocation_id) GlobalInvocationID : vec3<u32>) {
       assert(ctx !== null, 'Failed to get WebGPU context from canvas');
 
       let usage = GPUTextureUsage.COPY_DST | GPUTextureUsage.RENDER_ATTACHMENT;
```
Comment (austinEng) [inline src/webgpu/web_platform/reftests/canvas_complex.html.ts:699]: should we add COPY_DST and RENDER_ATTACHMENT only when needed as well?

**Review by austinEng — APPROVED** [review]

File: src/webgpu/web_platform/reftests/canvas_complex.html.ts (hunk 1 above)
Comment (jchen10) [reply src/webgpu/web_platform/reftests/canvas_complex.html.ts]: It would be better, done!

**Review by austinEng — APPROVED** [review]
looks great!

**Pull request merged**